packages/mobile/.gitignore

```diff
@@ -98,3 +98,6 @@ src/icons/DefaultAvatar.tsx
 src/images/*
 !src/images/Images.ts
 !src/images/link-arrow.png
+
+# E2E Test Secrets
+test/.env
```

Comment:
Looks like this one is not necessary, since it was added to `packages/mobile/e2e/.gitignore`

Reply:
Removed the unused line from the .gitignore in `packages/mobile/test/.gitignore`